### PR TITLE
Change the CNAME records for cfp domains

### PR DESCRIPTION
**Pull request by cfpwastaken**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel or Netlify.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.  

## Website Link/Preview

I only changed the CNAME to another domain on another host which doesn't have as much suction as noip.

### CODE REVIEW

File: domains/cfp.json [hunk 1]
```diff
@@ -5,6 +5,6 @@
 	  "discord": "cfp (318394797822050315)"
 	},
 	"record": {
-	  "CNAME": "cfp.gotdns.ch"
+	  "CNAME": "dyn.picoscratch.de"
```
Comment (Stef-00012) [inline domains/cfp.json:8]: I cannot reach your website
"404 page not found"

File: domains/pic.cfp.json [hunk 2]
```diff
@@ -6,6 +6,6 @@
 	  "discord": "cfp (318394797822050315)"
 	},
 	"record": {
-	  "CNAME": "cfp.gotdns.ch"
+	  "CNAME": "dyn.picoscratch.de"
```
Comment (Stef-00012) [inline domains/pic.cfp.json:9]: I cannot reach your website
"404 page not found"

File: domains/scfp.json [hunk 3]
```diff
@@ -6,6 +6,6 @@
 	  "discord": "cfp (318394797822050315)"
 	},
 	"record": {
-	  "CNAME": "cfp.gotdns.ch"
+	  "CNAME": "dyn.picoscratch.de"
```
Comment (Stef-00012) [inline domains/scfp.json:9]: I cannot reach your website
"404 page not found"

File: domains/status.cfp.json [hunk 4]
```diff
@@ -6,6 +6,6 @@
 		"discord": "cfp (318394797822050315)"
 	},
 	"record": {
-		"CNAME": "cfp.gotdns.ch"
+		"CNAME": "dyn.picoscratch.de"
```
Comment (Stef-00012) [inline domains/status.cfp.json:9]: I cannot reach your website
"404 page not found"

**Review by Stef-DP — CHANGES_REQUESTED** [review]

**Comment by cfpwastaken** [comment]
@Stef-00012 First of all, thanks for the quick review!
That domain is actually just used for my dynamic DNS and it points to a webserver that hosts multiple domains' websites at once and it can't guess which one you are trying to see when you just enter dyn.picoscratch.de. Please try entering the IP with domain into your hosts file and navigate directly to the domain. Since the current records point to the same IP in the end you can also check the current pages. They won't change.
Thanks!

**Comment by Stef-00012** [comment]
> @Stef-00012 First of all, thanks for the quick review! That domain is actually just used for my dynamic DNS and it points to a webserver that hosts multiple domains' websites at once and it can't guess which one you are trying to see when you just enter dyn.picoscratch.de. Please try entering the IP with domain into your hosts file and navigate directly to the domain. Since the current records point to the same IP in the end you can also check the current pages. They won't change. Thanks!

I see, after adding it to my hosts file, I'm still unable to access `scfp.is-a.dev` (nginx 502 bad gateway) and `status.cfp.is-a.dev` (The connection is not private, and since `.dev` domains require https I can't access it)

**Comment by cfpwastaken** [comment]
I have fixed the 2 pages now. Please check them again.
Take note that my IP address has changed by now to `87.150.220.123` (25.02.2024)

**Review by iostpa — APPROVED** [review]
website accessible now

**Pull request merged**